app/components/login_button_component.rb

```diff
@@ -0,0 +1,30 @@
+class LoginButtonComponent < ButtonComponent
+  APP_NAME = 'Login.gov'
+  VALID_COLORS = ['primary', 'primary-darker', 'primary-lighter'].freeze
+
+  attr_reader :color, :tag_options
+
+  def initialize(color: 'primary', **tag_options)
+    if !VALID_COLORS.include?(color)
+      raise ArgumentError, "`color` #{color}} is invalid, expected one of #{VALID_COLORS}"
+    end
+
+    super(
+      color: color,
```

Comment:
Somewhat related to the prior point about subclassing: `color` is not a property of `ButtonComponent`, so I wouldn't expect we'd be passing it this way. Currently, it's being passed through as an attribute of the `button` tag, which is unexpected:

```html
<button color="primary-darker" class="usa-button login-button login-button--primary-darker"> 
  Sign in with <span class="login-button__logo login-button__logo-white">Login.gov</span>
</button>
```